crates/turborepo-lib/src/run/scope/filter.rs

```diff
@@ -238,6 +238,7 @@ impl<'a, T: GitChangeDetector> FilterResolver<'a, T> {
             selectors.into_iter().partition::<Vec<_>, _>(|t| !t.exclude);
 
         let mut include = if !include_selectors.is_empty() {
+            // TODO: add telemetry for each selector
```

Comment:
just a placeholder for myself. Once we've parsed all the `--filter` flags, I'd really like to see more data on how people are using this (without any data for the values passed to the flag, just the "types" of filter)